TS3QueryLib.Net.Core/Server/Responses/ClientInfoBaseCommandResponse.cs

```diff
@@ -35,8 +36,9 @@ protected override void OnPostApplyResponse()
             MonthBytesDonwloaded = BodyCommandParameterGroupList.GetParameterValue<ulong>("client_month_bytes_downloaded");
             TotalBytesUploaded = BodyCommandParameterGroupList.GetParameterValue<ulong>("client_total_bytes_uploaded");
             TotalBytesDownloaded = BodyCommandParameterGroupList.GetParameterValue<ulong>("client_total_bytes_downloaded");
-            IconId = BodyCommandParameterGroupList.GetParameterValue<uint>("client_icon_id");
```

Comment:
was it replaced by client_flag_avatar? can you please point to a source describing the change?

Reply:
IconId caused an error, but I'm currently unable to point it out (because it doesn't occur)